src/os/bluestore/BlueStore.cc

```diff
@@ -11670,7 +11670,8 @@ void BlueStore::_kv_sync_thread()
       kv_sync_in_progress = false;
       kv_cond.wait(l);
       dout(20) << __func__ << " wake" << dendl;
-    } else {
```

Comment:
IIUC these 'else' removal makes a little sense - we just avoid a single loop iteration after above kv_cond.wait() terminates